src/pages/FilesPage/components/FolderItem/FolderItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Typography from '@mui/material/Typography';
+import {
+	FolderItemCard,
+	FolderItemCheckbox,
+	FolderItemContainer,
+	FolderItemIcon,
+} from './FolderItem.styles';
+import { FolderResponse } from 'clients/CoreService';
+import { renderLongName } from '../renderers/renderLongName';
+import { getBasename } from 'shared/util';
```

Comment:
```suggestion
import { basename } from 'path';
```
Use "path" lib instead